Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,39 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=True,
+            pool=False
+        )
+    )
+
+label monika_curse_words:
+    m 3etd "Say, {w=0.2}[player], {w=0.2}do you ever swear?"
+    m 1hksdlb "Ahaha, {w=0,2}I mean, {w=0,2}it's alright if you swear occasionally, {w=0.2}a lot of people do, after all."
+    m 2rtd "I even heard of some studies stating that using stronger language helps easing the pain if you, say, hurt yourself."
+    m 3eta "It's called the analgesic response, which makes your body more impervous to pain."
+    m 3esc "I know some people have it as natural part of their speech patterns, {w=0.2}and that your vocabulary is formed by society and those around you."
+    m 3efb "These expressions can make a talk feel a lot more casual, {w=0.2}and even help spice up certain conversations..."
+    m 3rksdlb "But there's also such a thing as too much swearing, I'd say"
+    m 2eud "Swearing can actually reveal a lot about our way of behaving in company of others."
+    m 3eub "For instance, {w=0.2}people are much less likely to swear in front of family than with friends, {w=0.2}and the more people you address at once, the more eloquently you try to phrase yourself."
+    m 3hksdlb "Of course, not everyone cares what other people think of the way they speak."
+    m 7eud "Personally, I prefer soft language over swearing in most situations.{w=0.3}{nw} "
+    extend 2etb "I feel it contributes to a more positive atmosphere overall, {w=0.2}and you don't have to adjust your speech as much depending on who you're talking to."
+    m 2ttb "And you know how certain people love atmosphere, {w=0.2}don't you [player]?"
+    m 3dkb "I think our language has so many graceful ways of expressing how you feel about this world, {w=0.2}and even the worst things could be spoken of with a bit of style and respect."
+    m 1eka "I'm not saying you always have to sound poetic when you speak, {w=0.5}{nw}"
+    extend 1hksdlb "of course not..."
+    m 3hksdlb "I'm just trying to say that I really like when people keep their more... {w=0.5}colorful language varied."
```

Comment:
```suggestion
    m 3hksdlb "I'm just trying to say that I really like when people keep their more... {w=0.3}colorful language varied."
```

Oh yeah, I forgot that we usually don't use w=0.5 these days, we mostly use 0.3 for flow reasons.

Reply:
Oh, okay, I just thought half a second fit for the ellipsis.

Reply:
When I started I thought so too, but after seeing both, I think generally 0.3 flows better.